keyboards/draytronics/elise_v2/keymaps/default/keymap.c

```diff
@@ -17,6 +17,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 #include QMK_KEYBOARD_H
 
+//Fix to prevent key repeat on wake from sleep when using a KVM / non direct usb connection.
+#undef USB_POLLING_INTERVAL_MS
+#define USB_POLLING_INTERVAL_MS 8
+#undef USB_SUSPEND_WAKEUP_DELAY
+#define USB_SUSPEND_WAKEUP_DELAY 3000
```

Comment:
These will not function when specified within this file. They need to be moved to a keymap level `config.h`.

Reply:
My apologies this is now resolved and I will push a commit with these changes.